.jenkins/pytorch/macos-build.sh

```diff
@@ -35,11 +35,11 @@ fi
 
 cross_compile_arm64() {
   # Cross compilation for arm64
-  USE_DISTRIBUTED=1 CMAKE_OSX_ARCHITECTURES=arm64 MACOSX_DEPLOYMENT_TARGET=11.0 USE_MKLDNN=OFF USE_QNNPACK=OFF WERROR=1 BUILD_TEST=OFF USE_PYTORCH_METAL=1 python setup.py bdist_wheel
+  CMAKE_OSX_ARCHITECTURES=arm64 MACOSX_DEPLOYMENT_TARGET=11.0 USE_MKLDNN=OFF USE_QNNPACK=OFF WERROR=1 BUILD_TEST=OFF USE_PYTORCH_METAL=1 python setup.py bdist_wheel
 }
 
 compile_x86_64() {
-  USE_DISTRIBUTED=1 WERROR=1 python setup.py bdist_wheel
+  WERROR=1 python setup.py bdist_wheel
```

Comment:
And keep this one in place
```suggestion
  USE_DISTRIBUTED=1 WERROR=1 python setup.py bdist_wheel
```

Reply:
Actually, I just realized that this doesn't work as-is. It looks like CI builds and tests x86, but only builds arm. This particular failure only shows up at runtime, not at build time, so wouldn't be caught.

Am I misreading the list of runs over in https://github.com/pytorch/pytorch/pull/86451/checks?check_run_id=8770362393

```
[macos-12-py3-x86-64-build](https://circleci.com/gh/pytorch/pytorch/17160240?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link) - Success
[macos-12-py3-x86-64-test-2-2-default](https://circleci.com/gh/pytorch/pytorch/17160360?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link) - Running
[macos-12-py3-x86-64-test-1-1-functorch](https://circleci.com/gh/pytorch/pytorch/17160361?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link) - Running
[macos-12-py3-x86-64-test-1-2-default](https://circleci.com/gh/pytorch/pytorch/17160362?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link) - Running
upload test status - Blocked
[ios-12-5-1-x86-64](https://circleci.com/gh/pytorch/pytorch/17160243?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link) - Success
[macos-12-py3-arm64-build](https://circleci.com/gh/pytorch/pytorch/17160244?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link) - Running
[ios-12-5-1-x86-64-coreml](https://circleci.com/gh/pytorch/pytorch/17160241?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link) - Success
[macos-12-py3-x86-64-lite-interpreter-build-test](https://circleci.com/gh/pytorch/pytorch/17160242?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link) - Success
```

Reply:
Just added this back in as a change, at least for the time being.